arabic/unpacked/html.js

```diff
@@ -0,0 +1,90 @@
+MathJax.Hub.Config({
+  'HTML-CSS': {
+    undefinedFamily: 'Amiri',
+    styles: {
+      '.mfliph': {
+        'display': 'inline-block !important',
+        '-moz-transform': 'scaleX(-1)',
+        '-webkit-transform': 'scaleX(-1)',
+        '-o-transform': 'scaleX(-1)',
+        'transform': 'scaleX(-1)',
+        '-ms-filter': 'fliph',
+        'filter': 'fliph'
+      },
+      '.mar': {
+        'font-family': 'Amiri !important',
+        'font-style': 'normal !important'
+      }
+    }
+  }
+});
+
+
+MathJax.Hub.Register.StartupHook('mml Jax Ready', function () {
+  MathJax.Hub.Register.StartupHook('HTML-CSS Jax Ready', function () {
+    MathJax.Hub.Register.StartupHook('Arabic TeX Ready', function () {
```

Comment:
- [ ] Thanks for the note. Will remove it. Will remove the extra signal-check.
 - [ ] I'll see about how does this affect `Arabic TeX Ready`.